src/app.js

```diff
@@ -183,12 +181,14 @@ function parse_weather_data(data) {
     var item = items[i];
     if (item.precipType !== undefined) {
       var prob = parseFloat(item.precipProbability);
+      var percentage = prob * 100;
```

Comment:
As far as I remember, the probability in the api goes to something like 4 decimal places, which even as a percentage is far too much information to a) be useful or b) fit on the screen nicely. Should we just round to an integer percentage?